code/modules/power/singularity/collector.dm

```diff
@@ -106,9 +106,9 @@
 				var/datum/bank_account/D = SSeconomy.get_dep_account(ACCOUNT_CAR)
 				if(D)
 					var/payout = output/8000
-					stored_power -= min(payout*200, stored_power)
-					payout = payout * balancevalue
-					D.adjust_money(payout)
+					stored_power -= min(payout*20000, stored_power)
```

Comment:
Wuat does this change do

Reply:
> Wuat does this change do

Puts it back to how it was before I touched it